tests/sentry/web/frontend/test_auth_channel_login.py

```diff
@@ -1,22 +1,26 @@
-from functools import cached_property
-
 from django.urls import reverse
 
+from sentry.auth.providers.fly.provider import FlyOAuth2Provider
+from sentry.models.authprovider import AuthProvider
 from sentry.testutils.cases import TestCase
 from sentry.testutils.silo import region_silo_test
 
 
 @region_silo_test
-class AuthOrganizationIdentifierLoginTest(TestCase):
-    @cached_property
-    def organization(self):
-        return self.create_organization(name="test org", owner=self.user)
-
-    @cached_property
-    def path(self):
-        return reverse("sentry-auth-organization-id", args=[self.organization.id])
+class AuthOrganizationChannelLoginTest(TestCase):
+    def setup(self):
+        self.organization = self.create_organization(name="test org", owner=self.user)
+        self.partner_org_id = "foobar"
+        config_data = FlyOAuth2Provider.build_config(resource={"id": self.partner_org_id})
+        self.auth_provider = AuthProvider.objects.create(
+            organization_id=self.organization.id, provider="fly", config=config_data
+        )
+        self.path = (
+            f"{reverse('sentry-auth-channel',args=['fly'])}?organization_id={self.partner_org_id}"
```

Comment:
does black not fix the whitespace here (in the reverse arguments) automaticallly or was it just not run?

Reply:
🤔 

what were you expecting in teh arguments here?
spaces between the args?

black was definitely run, so i guess it doesn't have an issue with the whitespaces here? 🤷